io.github.fkinoshita.Wildcard.json

```diff
@@ -0,0 +1,70 @@
+{
+    "id" : "io.github.fkinoshita.Wildcard",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "wildcard",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--socket=wayland"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin"
+    },
+    "cleanup" : [
+        "/include",
+        "/lib/pkgconfig",
+        "/man",
+        "/share/doc",
+        "/share/gtk-doc",
+        "/share/man",
+        "/share/pkgconfig",
+        "*.la",
+        "*.a"
+    ],
+    "modules" : [
+        {
+            "name": "blueprint-compiler",
+            "buildsystem": "meson",
+		    "cleanup": [
+			    "*"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://gitlab.gnome.org/jwestman/blueprint-compiler",
+                    "tag": "v0.8.1"
+                }
+            ]
+        },
+        {
+            "name" : "wildcard",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "build-options": {
+                "env": {
+                    "CARGO_HOME": "/run/build/wildcard/cargo"
+                }
+            },
+            "build-commands": [
+                "cargo --offline fetch --manifest-path Cargo.toml --verbose",
+                "cargo --offline build --release --verbose",
+                "install -Dm755 ./target/debug/wildcard -t /app/bin/"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/fkinoshita/Wildcard.git",
+		    	    "tag" : "0.1.0",
+		    	    "commit" : "1f05098c9aba1b9fd9626a46a70b6e48f1aca63c"
```

Comment:
I recommend generating a dist tarball and instead using it. You meson build system seems to support vendoring the crates. This would make 'cargo-sources' no needed.

Reply:
Okay, I'm running into some issues with meson, it's not finding blueprint-compiler with the `meson setup` command.